ui_framework/components/modal/modal.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export function KuiModal({ children, ...rest }) {
+  return (
+    <div className="kuiModal" { ...rest }>
```

Comment:
@Stacey-Gammon Not _sure_ what I think of this myself, but this was my thinking: My main "concern/thought" around wrappers is that if it's a DOM wrapper you know what can be in `rest` (any relevant html attribute), but if it's a React component that becomes a bit more unclear (it also makes the api unclear, as it's not super-clear what you actually expect to receive and pass down into lower-level components).

Also: React will let you know if DOM elements receives the wrong attribute (https://facebook.github.io/react/warnings/unknown-prop.html).

However, I also think there are edge-cases to this. E.g. on Cloud we had a couple different Button helpers that receive `rest` even if they weren't "pure DOM components" (they passed on to lower-level button helpers, e.g. `DangerButton` wrapped `SpinButton` that wrapped regular DOM buttons).

I think `rest` is a good approach if you're expected to see it as a "DOM component" from the outside (e.g. a `DangerButton` that might have some extra fields, but you can give it everything you can give a normal DOM `button`).

Historically I've added these `rest` params based on actual need in the code, but that might be a bit different in this context as 1) the team is much larger, 2) we're building a UI framework.